src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,85 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final int SUPPLY_VALUE_INDEX = 0;
+    private static final int BUY_VALUE_INDEX = 1;
+    private static final String BUY_NAME = "buy";
+    private static final String SUPPLY_NAME = "supply";
+    private static final String RESULT_NAME = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] readInfo = readFromFile(fromFileName);
+        String result = calculateResult(readInfo);
+        writeToFile(result, toFileName);
+    }
+
+    private int[] readFromFile(String fileName) {
```

Comment:
split this method into two, one for reading lines from file and the second one for calculation of buy, supply and result